### PR TITLE
Gcb hub updates 2024

### PR DESCRIPTION
**Pull request by nomadscientist**
Here we use the latest GCB/SIG tutorials in the GTN and combine this with the Hub to make for a more sustainable system to update stuff in both sites (and make it easy to send people links to FAQ when they want to make SIGs or join the GCB!)

### CODE REVIEW

**Comment by nomadscientist** [comment]
OK - this is ready for review & merging! It looks awesome!

**Review by hrhotz — APPROVED** [review]
looks good to me
Thank you very much Wendi

File: content/community/governance/gcb/index.md [hunk 1]
```diff
@@ -1,114 +1,63 @@
...
+This is analogous to the [Technical Board](/community/governance/gtb/), consisting of representatives of the [Working Groups (WG)](/community/wg).
 
-## Why make a SIG?
+You can learn more about **SIGs** in our dedicated tutorials below, or explore our [**SIG** directory](/community/sig).
 
-### Prevent work duplication
+<button type="button" onclick="window.open('https://training.galaxyproject.org/training-material/topics/community/tutorials/sig_define/tutorial.html', '_blank')" style="margin-right: 10px">
+  <img src="gtn_hat.png" alt="GTN Hat" width="50">
+  What is a SIG?
+</button>
```
Comment (hexylena) [inline content/community/governance/gcb/index.md:38]:
```suggestion
<a href="https://training.galaxyproject.org/training-material/topics/community/tutorials/sig_define/tutorial.html" style="margin-right: 10px">
  <img src="gtn_hat.png" alt="GTN Hat" width="50">
  What is a SIG?
</a>
```

would this work? I am uhh very against a button here. It's semanticlaly not a button. (don't want your a11y team to flag it later.)

Comment (nomadscientist) [reply content/community/governance/gcb/index.md]: Done!

**Pull request merged**